### PR TITLE
Removing farts by order of Pomf, and removing human vore.

### LINKED ISSUE #15504: Fat people can eat the nuke disk
#### Description of issue
If a fat person puts a nuke disk on a monkey and eats the monkey, the disk is stored in the fat person's container.

#### Steps to reproduce 
Be a fat person
Get the nuke disk
Get a monkey
Put the disk in the monkey's hand
Eat the monkey
You now have the disk in your body, which can only be removed through gibbing

#### Specific information for locating

I know @Intigracy was the one who worked on the revamp of eating people

#### Server revision

1ba76cf440e5295c2bc55da463f77cc4e424a01f 

#### Issue bingo

Please check whatever applies. More checkboxes checked increase your chances of not being yelled at by every contributor.

- [x] Issue could be reproduced at least once
- [ ] Issue could be reproduced by different players
- [x] Issue could be reproduced in multiple rounds
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] [Couldn't find an existing issue about this](https://github.com/d3athrow/vgstation13/issues)

### PR DESCRIPTION
**Pull request by N3X15**
# Executive Summary
Removes farts, and human vore by way of fatties eating monkeys and horrors eating people.  The eating mechanic still exists, but just gibs instead of the whole fetish fuel voring mechanic.

Xenos can still store people in their stomachs, although this may change in the future.

# Rationale
I know this is going to piss a lot of people off, so here's why:
Imgur

And on human vore:
Imgur

# Polling
 * Removing xeno vore was considered but server polling revealed that it was used tactically.
 * Removing fatties eating monkeys is being considered, although initial feedback is unfavorable.  Gibbing is offered instead.
 * Polling for removing farting and vore is unnecessary due to Pomf veto.

# Balancing
 * Digestion damage shouldn't really be needed in humans, since I removed the grab ingestion, but I bumped it up to reduce the fetish fuel component in case someone finds another way. Digestion should be 1/3rd of the duration before this adjustment.
 * Horrors are endgame and can pretty much instagib people via ingestion with this changeset, which may be a bit much.
 * Fatties eating monkeys is an old, old feature and was kept intact with the same mechanics as horrors.  Pomf may end up vetoing this anyway.

# Changelog
:cl:
 - rscdel: Fatties eating monkeys results in the gibbing of the monkey.
 - rscdel: Changeling horrors eating humans results in the gibbing of the human.
 - rscdel: The fart emote has been removed.
 - rscdel: Digesting creatures that do somehow make it into a carbon's stomach no longer results in farting, puking, and other stupid shit.  Digestion is silent and results in ghosting of the mob.
 - tweak: Digestion process deals 15 brute damage per SSair tick instead of 5.

# Fixes
* Fixes #15504

### CODE REVIEW

**Comment by unid15** [comment]
The madman absoluté

**Comment by Lokiamis** [comment]
I'm indifferent towards the eating stuff, but the farting is pretty stupid and iirc was actually added to /vg/ by a guy with a scat fetish

**Comment by small-boss** [comment]
here we go
but honestly, just because something can be connected with vore it doesn't have to be removed. you could connect a lot of things that happen ingame to various fetishes. but if pomf doesn't want it then it should go.

**Comment by Diavill** [comment]
*fart did nothing wrong

**Comment by N3X15** [comment]
The digestion farting was added by Intigracy, according to git blame.

no thank you

**Comment by Kammerjunk** [comment]
Yeah, Inti sounds like he'd be a fart fetishist to me

**Comment by Blithering** [comment]
Can you remove the creepy vore description you get when examining fat people?

**Comment by Riekland** [comment]
I kind of think that being able to forcefeed fat fucks chifir or something to make them puke up the monkey is good but I'm all for these changes still
death to vore

**Comment by Praecordia** [comment]
why remove horror form ling eating people, though?

**Comment by Orangerush** [comment]
fart stuff was added by some shitskin fart fetishist that you guys somehow let through, eating a monkey cube doesn't seem like vore stuff though, pretty funny because it fucks you up for being retarded

**Comment by Fruktlimpa** [comment]
Better remove alien devour then because that is vore :^)
Except it isn't, what the fuck is this nonsense? How can anyone consider eating a monkey cube vore?

**Comment by Shadowmech88** [comment]
I request that clowns specifically be able to fart.
Mostly because a clown's fart is a honk, allowing them to honk without the aid of a horn.

Alternatively add an emote for clowns that lets them honk their nose instead.

**Comment by ghost** [comment]
This server is supposed to have WACKY things though
Fucking retarded unless we start to enforce RP rules and change to medium RP .
>Le pomf vetos everything meme
MUH DEMOCRACY

**Comment by Kammerjunk** [comment]
What democracy?

**Comment by N3X15** [comment]
Testing complete.

**Comment by ghost** [comment]
farts are not funny, im not 9 anymore dad

**Comment by Orangerush** [comment]
'[..]im not 9 anymore dad. i'm above this' says the supreme gentlemen. 'i only like stuff that really makes me think about life....stuff like rick and morty'

**Comment by ghost** [comment]
Never thought I'd see the day

**Comment by Kammerjunk** [comment]
I dunno, farting humour sounds a lot like Rick & Morty humour

**Comment by Orangerush** [comment]
@Kammerjunk wubba lubba ping ling dinda dubb wubaa 
You're right i just wanted a reason to do a fedora meme

**Comment by Kammerjunk** [comment]
I'm looking at the original fart PRs right now and I'm not sure how it ever got this far

**Comment by Orangerush** [comment]
Stolen thread valor by posting this here. Funny stuff. How did you guys let this pass without realizing that he maybe gets off to the brap

**Comment by ghost** [comment]
>farting isn't funny

Neither was the German Nazi party OY VEEEY

**Comment by yclatious** [comment]
Eh, leave farting to the clown for shits and giggles, Im kinda neutral on the whole thing but leaving it in for the clownos just feels right instead of taking it away from them.

@Commandersand But making fun of the jews and nazis has always been fun, oy vey.

**Comment by Intigracy** [comment]
Xenos storing people to regurgitate later was never about vore and always about transporting people reliably to get facehugged / stuck to a bed in the nest.

I have no idea what I was thinking when I coded eating a monkey making you fart, and even less idea of why it got merged.

**Comment by ghost** [comment]
>caring about the fart fetishist

Who cares? Im sure we have at least one person who gets off to gore. Remove gibs when? Remove being able to be barefoot when?

**Comment by ghost** [comment]
the insidious argument now turns desperate

**Comment by Kammerjunk** [comment]
>caring about fetishies

Who cares? People can get off to anything, so there's no point in removing weird shit added specifically to fuel a fetish.
Roundstart non-valid tajaran when? ERP verbs when?

**Comment by ghost** [comment]
Psh, you don't know what you're dealing with kiddo, I complain about black people on the internet and share my advanced knowledge on /pol/ every day. You think you've won the argument? Guess again, im on multiple layers of irony therefore it is impossible to lose any argument as long as I state so.

**Comment by MusgardTiger** [comment]
Pushing somebody down and farting on their face is the ultimate powerplay.

**Comment by ghost** [comment]
No that's farting on their face before they can fart on yours.

Because fetish code is so quality, you can fart on people's face while you're stunned and they're standing.

**Comment by ghost** [comment]
You should add an annoying emote to replace farting though, sneezing on people  would be nice.

**Comment by Orangerush** [comment]
spitting is one of the ultimate insults there is, how about that instead of sneeze?

**Comment by Riekland** [comment]
rape is the ultimate insult, rape verbs from LW when

**Comment by Kammerjunk** [comment]
I mean, @Commandersand has a point.
We already have borgs for people who get off to doing as they're told, the FAT gene (>implying it's genetic) for fat fetishists, vox for furries, and floor tiles for that guy who jacked off to floor tiles

**Comment by N3X15** [comment]
@Kammerjunk doing a fine job at grasping for straws keep it up 👍

**Comment by Orangerush** [comment]
@N3X15 change farts to spitting on or at a person and bibles or I'm rioting

**Comment by Kammerjunk** [comment]
We can already look at naked sprites in-game so what's really the difference between that and having ERP verbs?
We should honestly just get ERP verbs

**Comment by JustSumBody** [comment]
put me in the screenshot

**Comment by RFultra** [comment]
What is erotic about farts
How could anyone get off to a girl sitting her giant ass on your face and forcing out a pungent fart straight into your face

**Comment by ghost** [comment]
Shouldn't Vox be removed if we're removing features because of a thread post? 🤔 
EDIT: Didn't even know that the vore flavor text even existed.

File: code/__HELPERS/globalaccess.dm [hunk 1]
```diff
@@ -29,20 +27,6 @@
 			return global.error_cooldown;
 		if("html_interfaces")
 			return global.html_interfaces;
-		if("all_lighting_corners")
```
Comment (JustSumBody) [inline code/__HELPERS/globalaccess.dm:32]: What all is going on in this entire file anyway

**Comment by chatsonf** [comment]
Well this fixes #15504

**Comment by Intigracy** [comment]
No it doesn't.

File: code/__HELPERS/globalaccess.dm (hunk 1 above)
Comment (N3X15) [reply code/__HELPERS/globalaccess.dm:32]: It's generated by a python script, see header.  I ran `python tools\gen_globals.py vgstation13.dme code\__HELPERS\globalaccess.dm` and that's what it came up with.

**Comment by d3athrow** [comment]
Just a note, my opinion doesn't necessarily mean law if the nerds want farting kept in then whatever
I still think it's stupid and something that should have stayed on goon.

**Comment by Lokiamis** [comment]
https://github.com/vgstation-coders/vgstation13/pull/1664

Reminder

**Comment by Kurfursten** [comment]
At least we don't have the sounds though. I'm indifferent to /vg/ farting but the goon farting sound effects are just dumb and constantly playing.

**Comment by d3athrow** [comment]
@Lokiamis Shh

**Comment by Dusk-a** [comment]
All I'm going to say is that instagibbing people is pretty brutal. If you want to get rid of any trace of vore shit you may as well remove the "eat anything" mutation. 
Fun fact: You can eat people's limbs with that but not their torso and head.

**Comment by chatsonf** [comment]
alright now this should fix #15504

File: code/modules/mob/living/carbon/alien/humanoid/life.dm [hunk 2]
```diff
@@ -455,6 +455,7 @@
 					M.death(0)
 					M.ghostize(1)
 					drop_stomach_contents()
+					M.drop_all()
```
Comment (Intigracy) [inline code/modules/mob/living/carbon/alien/humanoid/life.dm]: Shouldn't this be one line up?

**Comment by Intigracy** [comment]
@devlkiller0 The person eating a monkey doesn't gib, the monkey does.

Exception being eating a monkey cube, then you gib.

**Comment by Dusk-a** [comment]
I meant for the changeling. I know horror form is supposed to be scary and I don't know how long it takes for it to eat someone but it may be faster _and_ just gibs them so there isn't a chance to bring them back.

**Comment by N3X15** [comment]
Eating someone takes 10 seconds in both iterations.

**Comment by TheFriggenSawbones** [comment]
are we becoming some unholy parody of ourselves

**Comment by N3X15** [comment]
We're just going another level of irony.

**Comment by Intigracy** [comment]
Gibbing spawns a head with brain, buddy.

**Comment by N3X15** [comment]
Adding [Balance] since this affects xeno, monkey, ling, and human combat.

**Comment by TheFriggenSawbones** [comment]
o u t
o f
t o u c h

**Comment by Riekland** [comment]
> TheFriggenSawbones commented 6 minutes ago

**Comment by CptWad** [comment]
this keeps the superfart gene right
its the most powerful power
also theres that fart defect that gives you space drugs if you fart for whatever reason
tbh, theres a lot of fetishists about and we should not code around trying to avoid them otherwise we will end up witch hunting and removing otherwise fun gameplay mechanics just because some people get off to it
I mean, there really is a guy that exists who finds floortiles atrractive and jerks off to it. He may not play space but that should not qualify removing floortiles.
Also @N3X15 if you have the time to remove features you should at least replace them with something of similar value
I suggest porting pies actually coating someones face when hit with said pie at least. I know they have it on other servers.

**Comment by Riekland** [comment]
>you have to put something in to take it out

actually piss off you fucking nematode

**Comment by CptWad** [comment]
I mean I am all for removing shit like mechanic because mechanic literally has no reason to exist
but the clowns holy right to fart in peoples face must be replaced with a new form of clown fu

**Comment by ghost** [comment]
If you add spitting make vox say "tfu tfu" when they spit
And plasmamen spit fire like they about to drop their mixtape on the station

**Comment by cadaverouscoyote** [comment]
when did we get so thin-skinned that the thought of somebody's fetish being made a mockery of became too icky for us to handle

and if anybody actually does pop a boner from watching a monkey disappear next to a sprite of a 250lb clown or having an a. wyatt mann cartoon in a grey jumpsuit blast ass in their face after they slip on a banana peel, they're the joke and they know it

calm down spergs

**Comment by CptWad** [comment]
hold the fucking bone
\>instead of the cooldown of swallowing people whole via changeling horrorform, you just need to grabhold someone then click them on you to I N S T A N T L Y gib them
wew

**Comment by TheFriggenSawbones** [comment]
god bless nexis knows what he's doing

**Comment by TheFriggenSawbones** [comment]
tfw you've been bugging nexis to fix antag prefs reseting, giving him a way to reproduce it

tfw he decides to autism away farting and "human vore"

what a great guy

**Comment by ghost** [comment]
I fully support this PR and the removal of all of this lecherous fetish fuel.  While we are at it, I personally have a chloral hydrate autoinjector fetish that just gets me a little too hot and bothered.  If you could go ahead and remove the ability for autoinjectors to have chloral hydrate in them, I will be able to better stay pure in mind and heart.

I am also incredibly grateful that you are giving my favorite antag-type the ability to instantly gib people due to the off chance that someone might otherwise start jerking off to game mechanics.  A perfectly rational and intense fear of people jerking off is an excellent basis to balance.

**Comment by IAGFPRMF** [comment]
Is this specifically targeted against things that are vore-like? Honestly I wouldn't make a connection to vore from a clown going "FEED ME" and then eating all the monkeys in genetics, that's just fucking funny. I almost forget it's a feature half the time, too. On the part of the aliens, it wouldn't really matter. I think I'm the only person who cuts off feet and swallow people instead of disarm spamming them anyway. 

Honestly we don't have to have a big fit over fear of "fetish fuel" because it's so vague and simple that I'd hardly call it fetish fuel.

The only big problem I have with this PR is my farts being taken away. MUH FARTS!

**Comment by carshalash** [comment]
It is a clowns right to be able to slip people and fart on them.

**Comment by FudgePucker** [comment]
Should we remove antags because somebody gets a murderboner?

**Comment by Chucklemonger** [comment]
if you remove farting then come up with some new genetics powers because more than one is related to that

**Comment by TheManNamedTom** [comment]
I can't believe that you suckers are thin-skinned enough to actually consider this. Who even cares, how many rounds are ruined by fuckin' farts? Why can you wear kinky underwear and go commando, but the big issue is eating monkeys and farting? There are so many more important issues than this that people actually want dealt with, but nah lets remove a harmless funny feature because an ice cream eating chicken can't handle it, who cares if goes against the will of the majority of the playerbase?
Better remove soap, the Chain of Command, slime toxin, catbeasts, butts, photocopying asses; hell, it doesn't matter what it is, someone somewhere will fetishize it, better just drop the whole game to be safe.

*fart

**Comment by Goldsteel** [comment]
NOOOOOOOOOOOOOOO ITS NOT FAIR HOW WILL I BULLY SECURITY NOW WITHOUT PUSH/FART?

**Comment by IAGFPRMF** [comment]
where the fuck is the drama tag by now smh

**Comment by nuklearcellphoneg** [comment]
@Goldsteel 

By wrongfully banning them of course.

**Comment by Goldsteel** [comment]
Whew

**Comment by strawberrytan** [comment]
>vore a hos as xeno, bring him in for hugging
>A few hours later eating mechanics removed, xenos next
The metaclub is real

**Comment by ghost** [comment]
Last time someone added/removed a genetics power, the entire fucking thing imploded. Did you t e s t it

**Comment by N3X15** [comment]
yes

**Comment by Exxion** [comment]
@IAGFPRMF I deleted it because it did nothing but make drama worse
Also I'm not sure whether this is the best or the worst comment thread on the GitHub

**Comment by DamianX** [comment]
as a follow up, when are we removing buttbots?

**Comment by Kammerjunk** [comment]
@cadaverouscoyote Yeah haha
#828 #884 #1049 
Sure LAUGHED AT HIM hard

**Comment by carshalash** [comment]
image

**Comment by ghost** [comment]
nothing stops you from custom emoting farts.

the OP doesn't suggest it's being removed because it's a fetish, it's being remove because pomf said so.

what does *fart actually bring to the server? in my opinion the first slip and fart is the same as the last 5000, it's never made me laugh or think or care. surely a 4chan server should be above a fart joke.

**Comment by Diavill** [comment]
but pomf didnt "order" farting to be removed

**Comment by SkowronX** [comment]
👌

**Comment by ghost** [comment]
Idea: replace farting with burping

**Comment by Ephx1** [comment]
So we're removing vox then too, right?

**Comment by ghost** [comment]
Remove ayy, humans only and lower tick rate so we can have ss13 XCOM
everyone gets ablative to emulate XCOM evasion

**Comment by ghost** [comment]
Don't we have literal gay bondage gear in the autodrobe and a library full of actual fetish porn?

**Comment by ghost** [comment]
>gay bondage gear

Fuck you
mqdefault

**Comment by strawberrytan** [comment]
Everybody has a Homer Simpson burp on a fart cooldown instead

**Comment by mtpowell** [comment]
Cancerous PR, the chicken is wrong

**Comment by Arthurdentist** [comment]
will this be more or less controversial then hypothermia? 

Hypothermia has the lead but there is still plenty to play for, who is it going to be? Stay tuned for more!

**Comment by ghost** [comment]
I think before we consider this we need to get our perspectives right.

**Imagine for a minute that you're Pomf.**
Now look at this: 
firefox_2017-08-30_16-44-51
mspaint_2017-08-30_16-59-41
That's just wrong. That isn't just "something someone might get off to", that's clearly some sick fuck having his way with your server and getting away with it.
How would YOU feel if this was your server? I'm sure you can understand why Pomf feels the way he does about the addition of farts, even if you enjoy farts yourself. I sure as hell would want to right that wrong, too.

Now about vore:
Digestion is super fucking cringy.
firefox_2017-08-30_16-40-10
That obviously should be tuned down. As for the rest of the changes, I don't think anyone really feels uncomfortable with a cartoon fat guy eating an NPC monkey. 
Remember, we shouldn't witchhunt content because "someone might get off to it", we should remove shit that was clearly only added for personal fetish fuel. I think both images count. But let's not get prudish over random shit.

**Comment by Orangerush** [comment]
pomf probably "ordered" this off the cuff after seeing those thread posts tbh

**Comment by Shadowmech88** [comment]
>we should remove shit that was clearly only added for personal fetish fuel

Even if, hypothetically, those features are enjoyed by people other than the weird fetish guy who added them, for reasons not fetish-related?

**Comment by ghost** [comment]
Ask yourself if the whole fetish injection shit would fly nowadays, the PR would be mocked and shut down, even if a couple people would enjoy it. No difference.
My answer is yes, out of principle.

**Comment by Riekland** [comment]
oh god what is that vore
I thought it just made you do the *fart command not gave you special brap haha ;)))))))) lines

**Comment by ghost** [comment]
If I fapped to datums involving ss13 antags can we hault all antag datum progress

**Comment by Kammerjunk** [comment]
Vore farts are an audible clue that a person has been consumed
Can't remove it now you fuckers

**Comment by ghost** [comment]
LAWYER'D

**Comment by carshalash** [comment]
Remove floor tiles from because of that one guy.

**Comment by Killette2** [comment]
Can we remove forcefeeding? I'm afraid that someone might become inappropriately aroused with a feeder fetish. This also makes that rape at worst to the non-consenting victim, or at best it's still ERP with consent.

**Comment by N3X15** [comment]
Muh slippy slopes

**Comment by Ephx1** [comment]
This is actually sticky slopes

**Comment by RFultra** [comment]
cluwning is a fetish, remove it

**Comment by Boxe** [comment]
heredth was a mistake

**Comment by N3X15** [comment]
Going to split this into two seperate PRs.

**Comment by NigglyWiggly** [comment]
Remove Heredth REMOVE HEREDTH

**Pull request closed without merging**